spec/integration/knife/cookbook_upload_spec.rb

```diff
@@ -97,5 +97,31 @@
         expect { knife("cookbook upload x -o #{cb_dir}") }.to raise_error(Chef::Exceptions::MetadataNotValid)
       end
     end
+    
+    when_the_repository "have cookbooks at muliple paths" do
```

Comment:
```suggestion
    when_the_repository "has cookbooks at multiple paths" do
```

Reply:
@phiggins Agreed, I updated the changes.